modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/util.js

```diff
@@ -571,8 +571,11 @@
 			return Util.getTop().Liferay.Util.Window.getById(id);
 		},
 
+		/**
+		 * @deprecated As of Athanasius (7.3.x), replaced by `window.innerWidth`
+		 */
 		getWindowName() {
-			return window.name || Window._name || '';
```

Comment:
Also, the more I think about it, we can probably just leave this code in and deprecate the function. Removing this bit of code doesn't really do anything for us since we no longer depend on the util internally. The whole thing will be removed eventually after deprecation anyways